packages/create-emotion/src/index.js

```diff
@@ -259,7 +259,17 @@ function createEmotion(
       identifierName += `-${p1}`
       return ''
     })
-    name = hashString(styles + identifierName) + identifierName
+    if (process.env.NODE_ENV !== 'production') {
+      name =
```

Comment:
Does this problem affect production env only? I suspect it does not and any solution for this should be applied regardless of the environment

Reply:
Oh, after investigating the source code some more it seems that source maps are handled only in non-production code anyway, so this check here is OK. I would hoist `sourceMapRegEx` up and reuse it at both places.

Reply:
One has the global flag and the other has not. I could however do something like this:
```js
const sourceMapRegEx = /\/\*#\ssourceMappingURL=data:application\/json;\S+\s+\*\//
const sourceMapRegExGlobal = new RegExp(sourceMapRegEx, 'g')
```
Or do you have another suggestion?